mysql-test/suite/tianmu/t/create_db.test

```diff
@@ -0,0 +1,7 @@
+#
```

Comment:
Check whether the Tianmu engine is ready before testing.
Like:
<img width="374" alt="image" src="https://user-images.githubusercontent.com/106147765/190299040-c2ddd33d-e34e-49ca-8310-f16b45051b3d.png">